Entity/CategoryManager.php

```diff
@@ -97,6 +97,36 @@ public function getSubCategoriesPager($categoryId, $page = 1, $limit = 25, $crit
     }
 
     /**
+     * NEXT_MAJOR.
```

Comment:
I hope this note `NEXT_MAJOR: keep this method.` will be enough, because I dont know what else to add there.